src/browser/api/window.js

```diff
@@ -517,6 +517,16 @@ Window.create = function(id, opts) {
             emitToAppAndWin('crashed', 'out-of-memory');
         });
 
+        browserWindow.on('blur', () => {
```

Comment:
Instead of doing this here, can you add decorators to the `focus` and `blur` events and move this code there:

https://github.com/HadoukenIO/core/blob/develop/src/browser/api/window.js#L66
https://github.com/HadoukenIO/core/blob/develop/src/browser/api/window.js#L528

Reply:
code moved per request updated test results

* [Windows 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/59d25afe7cb79f732d10c41d)
* [Windows 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/59d25b5f7cb79f732d10c41e)